Formula/mmctl.rb

```diff
@@ -2,10 +2,9 @@ class Mmctl < Formula
   desc "Remote CLI tool for Mattermost server"
   homepage "https://github.com/mattermost/mmctl"
   url "https://github.com/mattermost/mmctl.git",
-      tag:      "v5.26.0",
+      tag:      "v5.27.0",
       revision: "1dc9cda2e80757ff8fc694f32efc7ea76e327635"
```

Comment:
This revision does not change across the tag issue is [a known issue](https://github.com/mattermost/mmctl/issues/155).